runner/org.eclipse.rcptt.runner/META-INF/MANIFEST.MF

```diff
@@ -35,9 +35,9 @@ Require-Bundle: org.eclipse.rcptt.core,
  org.eclipse.rcptt.reporting.util;bundle-version="[2.5.5,3.0.0)",
  org.eclipse.rcptt.launching.rap;bundle-version="[2.5.5,3.0.0)",
  org.eclipse.swt,
- org.bouncycastle.bcpg;bundle-version="1.70.0",
  org.slf4j.api;bundle-version="1.7.30",
- org.bouncycastle.bcprov;bundle-version="1.70.0",
- jakarta.servlet-api;bundle-version="4.0.0"
+ jakarta.servlet-api;bundle-version="4.0.0",
```

Comment:
Using `Import-Package` instead of `Require-Bundle` would be more flexible and therefore better.

Reply:
Do not use Import-Package in Eclipse plugins (at all, never). Their flexibility and poor support for split-packages, lead to errors that are not detectable at design time and appear only at runtime (in particular, dependency loop detection often fails when Import-Pakage is used).

Reply:
@HannesWell I followed your hint at https://github.com/eclipse/chemclipse/pull/1627, but I won't here to comply with maintainers preferences.